content/workers/platform/routing/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+pcx-content-type: concept
+title: Routing
+---
+
+# Routing
+
+## Background
+
+Routing lets customers connect a Worker to the Internet, allowing it to receive HTTP requests on their Cloudflare zones. There are two ways to route to a Worker: [Custom Domains](/workers/platform/routing/custom-domains) and [Routes](/workers/platform/routing/routes).
+
```

Comment:
From Kirk: Towards the beginning, I think we need something like "Routes adds workers functionality to your existing Orange-clouded hostnames, as a proxy in front of your Application server. Custom Domains can replace the Orange-cloud process entirely - automatically adding a worker and an SSL cert to a given hostname"